oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/storage/model/StorageModels.java

```diff
@@ -113,6 +117,13 @@ private void retrieval(final Class<?> clazz,
 
         for (Field field : fields) {
             if (field.isAnnotationPresent(Column.class)) {
+                if (field.isAnnotationPresent(SQLDatabase.AdditionalEntity.OnlyAdditional.class)
+                    || field.isAnnotationPresent(SQLDatabase.AdditionalEntity.OriginAndAdditional.class)) {
```

Comment:
I think we need to check `at most one` for `OnlyAdditional` and `OriginAndAdditional`.
Both existing seems a conflict in the following logic.